python/hsfs/feature_view.py

```diff
@@ -219,10 +219,10 @@ def get_feature_vectors(
 
     def get_batch_data(self, start_time=None, end_time=None, read_options=None):
```

Comment:
There is a also a `get_batch_query` function which takes date arguments that has not been updated

`def get_batch_query(
        self, start_time: Optional[datetime] = None, end_time: Optional[datetime] = None
    )`

Reply:
that method call `get_batch_query_string` and that method uses `convert_event_time_to_timestamp`